### PR TITLE
Add gnocchi-injector

### PR DESCRIPTION
**Pull request by jd**
This allows to inject measures and maybe process them in a single tool. This is
useful to generate load to test metricd and also profile the processing code.

### CODE REVIEW

**Comment by jd** [comment]
Fix pep8

**Comment by jd** [comment]
Rebased

**Comment by jd** [comment]
Fix Swift

**Comment by jd** [comment]
Add release note

File: gnocchi/cli/injector.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                for s in inc.iter_sacks():
+                    c.process_new_measures_for_sack(s, blocking=True)
+            LOG.info("Processed %d sacks in %.2fs",
+                     inc.NUM_SACKS, sw.elapsed())
+            LOG.info("Speed: %.2f measures/s",
+                     float(total_measures) / sw.elapsed())
+
+        if interval is None:
+            break
+        time.sleep(max(0, interval - interval_timer.elapsed()))
```
Comment (chungg) [inline gnocchi/cli/injector.py]: does this loop forever until you ctrl+c?

Comment (jd) [reply gnocchi/cli/injector.py]: If you set an interval, yes.

**Pull request merged**